frontend/src/screens/wallet/send/ConfirmPayment.tsx

```diff
@@ -30,6 +31,9 @@ export default function ConfirmPayment() {
           method: "POST",
           body: JSON.stringify({
             amount: amount ? amount * 1000 : undefined,
+            metadata: identifier
+              ? { recipient_data: { identifier } }
```

Comment:
should we include the comment here too? e.g. paying to a WoS address we would not see the comment by default

Reply:
Resolving, discussed above^

Reply:
Can you review this again?

Reply:
We should set the comment here otherwise we rely on the recipient's lightning address server to set the memo on the invoice. At least one large lightning address provider does not do this.